_docs/server/configuration.md

```diff
@@ -12,39 +12,71 @@ This part of the documentation will focus on the configuration of the server. St
 
 To quickly open the config:
 
-```
+```shell
 $ lounge config
 ```
 
 ## public
 
-This setting can be either `true` or `false`. Public servers require no authentication. This is the default mode. Set to `false` to enable user login.
+This setting can be either `true` or `false`. Public servers require 
+no authentication. 
+This is the default mode. Set to `false` to enable user login.
+
+Example :
+```javascript
+public: true,
+```
 
 ## host
 
-Listen to connections only from this host. Default `0.0.0.0` will allow connections from anyone.
+Listen to connections only from this host. Default `undefined` will allow 
+connections from anyone (similar to `0.0.0.0`). 
+
+Examples :
+```javascript
+host: "127.0.0.1",
+host: undefined,
+```
 
 ## port
 
-The port to listen on.
+The port to listen on. Default to `9000`.
 
 You can override this setting by starting The Lounge like this:  
-`lounge start --port 80`
+`lounge start -- --port 80`
```

Comment:
Yes I just wrote it in order to avoid people coming and opening tickets all the time about this.